Artifacts/windows-powershell3/install.cmd

```diff
@@ -0,0 +1,14 @@
+@echo OFF
```

Comment:
Have you considered doing this in PS instead of a batch file? You would have more control. Also, is there a way to download the .MSU files instead of making them part of the artifact? That way you only download what is needed and we can also ensure it is the latest.